package.json

```diff
@@ -5,7 +5,7 @@
     "license": "Apache-2.0",
     "scripts": {
       "start": "vite",
-      "build": "vite build",
+      "build": "vite build --base=/bpmn-visualization-tutorial-getting-started/",
```

Comment:
⚠️ The `--base` option prevents to test locally.
Can we add a `build-local` script without this option and document in the README how to test the production build locally?

Reply:
done in 63e67f05